blockstack/blockstack-browser
# #1205 Upgrade to React 16

Addresses ticket #1205 

Changes required to upgrade to React 16:

- Updated versions on `react`, `react-dom`, and `react-modal`
- PropTypes has been moved out of React core into a standalone module
- The `react-radio-group` module does not support React 16, and a [simple PR](https://github.com/chenglou/react-radio-group/pull/45) opened by a community member in November 2017 is yet to be merged, which tells me this module is unmaintained. But that's okay since the only place we used this module actually didn't require or use it's functionality.

To test:
This ticket should require a full regression test to make sure no edge case functionality depends on non-Fiber-friendly code. I did as much initial testing as I could but this could use some more eyes.
